catroid/src/main/java/org/catrobat/catroid/pocketmusic/ScrollController.java

```diff
@@ -33,58 +33,75 @@
 
 import org.catrobat.catroid.R;
 import org.catrobat.catroid.pocketmusic.note.NoteLength;
+import org.catrobat.catroid.pocketmusic.ui.AnimatorUpdateCallback;
+import org.catrobat.catroid.pocketmusic.ui.PianoView;
 import org.catrobat.catroid.pocketmusic.ui.TactScrollRecyclerView;
+import org.catrobat.catroid.pocketmusic.ui.TrackRowView;
 
 public class ScrollController {
 
-	private static final int LAST_NOTE_IN_TRACK_VIEW = 8;
+	private static final int NUMBER_OF_NOTES_IN_TRACK_VIEW = 8;
```

Comment:
shouldn't it be dependend on the TACTS_PER_SCREEN variable from TactScrollRecyclerView?

Reply:
you are correct, made it now dependent on TACTS_PER_SCREEN and the MusicalBeat of the TrackGrid in the TactScrollRecyclerView

Reply:
great!